### PR TITLE
Pull request for Portfolio Project

### PR DESCRIPTION
**Pull request by GraceAbike**
For this repository, I've created HTML templates and styling files in order to build a responsive portfolio,  which can help to easily explore profile different devices.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <a href="#" class="connect">Let's Connect</a>
+      <ul>
+        <li><a href="https://twitter.com/GraceAbike1"><img src="./images/Twitt.png" alt=""></a></li>
+        <li><a href="https://www.linkedin.com/in/grace-abike-02770522a/"><img src="./images/Linkedin-icon.png" alt=""></a></li>
+        <li><a href="https://angel.co/u/grace-abike"><img src="./images/Min.png" alt=""></a></li>
+        <li><a href="https://github.com/GraceAbike"><img src="./images/Git.png" alt=""></a></li>
+        <li><a href="#"><img src="./images/Minimax.png" alt=""></a></li>
+      </ul>
+    </div>
+  </section>
```
Comment (Bondok6) [inline index.html:37]: - [ ] I noticed that this section has more space on the right side, please fix it. Remember you should stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

| Your Design  | Figma Design  |   
| --- | --- |
|    |  |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+ <section class="landing-page">
+    <div class="landing-text">
+      <h1>I'm Grace.<br>Glad To See You!</h1>
+      <p>I'm a software developer! I can help you build a product , 
+        feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+    </div>
+    <div class="social-link">
+      <a href="#" class="connect">Let's Connect</a>
```
Comment (Bondok6) [inline index.html:28]: - [ ] According to the requirements, You should Implement the button interactions (enable, hover, etc.). so I suggest adding a hove on the `connect` class.

For example
```
a.connect:hover {
  color: #fff;
  background-color: #7f8cff;
}
```

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="https://use.fontawesome.com/releses/v5.15.4/css/all.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header> 
+    <nav class="logo">
+     <a href="#">Grace Abike</a>  
+    <img src="images/Menu.png" alt="">
+  </nav>
+    <nav class="des"></nav>
```
Comment (Bondok6) [inline index.html:17]: - [OPTIONAL] I suggest deleting this line (I think it is not necessary) to make the code clean.

File: test.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+"This test file should ignore by git"
```
Comment (Bondok6) [inline test.yml:1]: - [OPTIONAL] I suggest deleting this file (I think it is not necessary) to make the code clean.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releses/v5.15.4/css/all.css">
+    <title>Portfolio</title>
+</head>
+<body>
+  <header> 
+    <nav class="logo">
+     <a href="#">Grace Abike</a>  
+    <img src="images/Menu.png" alt="">
+  </nav>
```
Comment (Bondok6) [inline index.html:16]: - [ ] Great job for using the semantic HTML tags correctly 💯. However, according to the requirements, You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). I noticed that the alignment between the logo and the menu icon is not the same as the Figma design, kindly fix it.

| Your Design  | Figma Design  |   
| --- | --- |
|    |  |

**Review by Bondok6 — CHANGES_REQUESTED** [review]
## Hi @GraceAbike🙋‍♂️ , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters are passing ✅
- Good commit history 💯✅
- Great job on implementing the project ✅💯

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (GraceAbike) [reply index.html]: thank you very much, I have made the changes

**Review by DammyShittu — APPROVED** [review]
Hi @GraceAbike,

## APPROVED 🟢

✔️ Nice styling

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GraceAbike** [comment]
Thanks to you all, I'm happy to complete the project.

**Pull request merged**